packages/common/src/ui/layout/tables/react-table-config.d.ts

```diff
@@ -0,0 +1,26 @@
+import {
```

Comment:
another TS issue. Had to add this to get the linter to smile. And then had to duplicate in `host` to get the pre-commit linting hook to allow me to commit. Need to see if I can get around that